src/aiohttp_middlewares/annotations.py

```diff
@@ -19,9 +21,15 @@
 )
 
 from aiohttp import web
-from aiohttp.web_middlewares import _Middleware as Middleware
 from yarl import URL
 
+try:
+   # (<3.9.0) Try to import Middleware from aiohttp.web_middlewares
+   Middleware = importlib.import_module('aiohttp.web_middlewares')._Middleware
+except AttributeError:
+   # (>=3.9.0) If that fails, import Middleware from aiohttp.typedefs
+   Middleware = importlib.import_module('aiohttp.typedefs').Middleware
+
```

Comment:
I think this needs one more space of indentation to pass [this test](https://github.com/playpauseandstop/aiohttp-middlewares/actions/runs/6942334701/job/18959911574?pr=185#step:4:270).

```suggestion
try:
    # (<3.9.0) Try to import Middleware from aiohttp.web_middlewares
    Middleware = importlib.import_module('aiohttp.web_middlewares')._Middleware
except AttributeError:
    # (>=3.9.0) If that fails, import Middleware from aiohttp.typedefs
    Middleware = importlib.import_module('aiohttp.typedefs').Middleware
```